extensions/shared/register-jetpack-block.js

```diff
@@ -19,6 +21,13 @@ function requiresPlan( unavailableReason, details ) {
 	return false;
 }
 
+// Injecting the `has-warning` class into the block wrapper component gives us
+// the right kind of borders around the block, both visually and conceptually.
+const withHasWarningClassName = createHigherOrderComponent(
+	BlockListBlock => props => <BlockListBlock { ...props } className="has-warning" />,
```

Comment:
This will need block-name check before applying the class.

Reply:
:+1: dc630d9